doc/specification.md

```diff
@@ -17,17 +17,10 @@ an SDN controller.
 This document serves as a specification for the gRIBI protocol.
 
 # 2 Data Model
-Uses AFT OC model.
 
-## 2.1 `NextHopGroup`
+gRIBI uses the [OC (OpenConfig) AFT model](https://github.com/openconfig/public/tree/master/release/models/aft) as an abstracted view of the device RIB. Using the same schema as the OC AFT model simplifies gRIBI injection service as much as possible. It guarantees that injected gRIBI entries are mappable to the existing gNMI `Get` and `Subscribe` RPCs for retrieving and streaming AFT entries.
```

Comment:
(nit) can we wrap to 80-characters? it makes things a bunch easier to read/review :-)

Reply:
ack, I'll file another PR to wrap everything once I merged all the current PRs.